### PR TITLE
Patient controller, model and service updates Features/9348

### PR DESCRIPTION
**Pull request by FuriousLlama**
# Fixes or Implements [AB#9348](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9348)

* [x] Enhancement
* [ ] Bug
* [ ] Documentation

## Description

Updated the patient model to retrieve gender. Added a new API to retrieve the patient using the phn

## Testing

* [x] Unit Tests Updated
* [ ] Functional Tests Updated
* [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

NO

### CODE REVIEW

File: Apps/Common/src/Models/PatientModel.cs [hunk 1]
```diff
@@ -22,37 +22,19 @@ namespace HealthGateway.Common.Models
...
-        /// <param name="email">The patient email.</param>
-        public Patient(string hdid, string phn, string firstName, string lastName, DateTime birthDate, string email)
-        {
-            this.HdId = hdid;
-            this.PersonalHealthNumber = phn;
-            this.FirstName = firstName;
-            this.LastName = lastName;
-            this.Birthdate = birthDate;
-            this.EmailAddress = email;
+            this.Gender = string.Empty;
```
Comment (leobcgov) [inline Apps/Common/src/Models/PatientModel.cs:37]: Is it possible for us to convert this to an enum?

**Review by leobcgov — APPROVED** [review]
good stuffs, I have a comment for improvement

File: Apps/Common/src/Models/PatientModel.cs (hunk 1 above)
Comment (FuriousLlama) [reply Apps/Common/src/Models/PatientModel.cs:37]: We could codify it if needed

Comment (leobcgov) [reply Apps/Common/src/Models/PatientModel.cs:37]: I think it should be fine for now as it comes from the source as a string and we just display it to our vue.

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**